pkg/cli/info.go

```diff
@@ -52,11 +57,27 @@ func (s *Info) Run(cmd *cobra.Command, args []string) error {
 		logrus.Errorf("failed to read CLI config: %v", err)
 		cfg = nil
 	}
-	projectInfo := make(map[string]apiv1.InfoSpec)
 
+	projectInfo := make(map[string]any, len(info))
 	// Format data from info response into map of project name to info response
 	for _, subInfo := range info {
-		projectInfo[subInfo.Name] = subInfo.Spec
+		if s.Verbose {
+			// If the user passed the verbose flag, then give them the entire info response
+			projectInfo[subInfo.Name] = subInfo.Regions
+			continue
+		}
+
+		// Remove unset fields in the config and userConfig
+		regionInfo := make(map[string]any, len(subInfo.Regions))
+		for region, spec := range subInfo.Regions {
+			specMap, err := removeUnsetFields(spec, "config", "userConfig")
```

Comment:
It feels like removing unset fields is a general formatting problem that can be solved at the `table.Writer` level; either by exposing some first class feature of `table.Writer` or by creating a new delegating writer type that proxies write calls, dropping the fields before passing the object along.

Reply:
The issue is that the table writer uses json/yaml marshaling. That means that in order to get the table writer to do this, we would have to use `omitempty`. However, we cannot do that with the config fields (see the comment on the Config type).

This implantation is essentially doing the "proxy" idea. Do you think it is worth it to try to make this more general right now?

Reply:
> Do you think it is worth it to try to make this more general right now?

No, I don't think so. I imagine we'd only pursue this if/when we need similar behavior in other commands.